instrumentation/aws_sdk/README.md

```diff
@@ -32,6 +32,31 @@ OpenTelemetry::SDK.configure do |c|
   c.use_all
 end
 ```
+### Configuration options
+This instrumentation offers the following configuration options: 
+* `:inject_messaging_context` (default: `false`): When set to `true`, adds context key/value 
+ to Message Attributes for SQS/SNS messages.
+* `suppress_internal_instrumentation` (default: `false`): When set to `true`, any spans with 
+ span kind of `internal` are suppressed from traces.
+
+## Integration with SDK V3's Telemetry support
+AWS SDK for Ruby V3 added support for Observability which includes a configuration, 
+`telemetry_provider` and an OpenTelemetry-based telemetry provider. Only applies to
+AWS service gems released after 2024-09-03. 
+
+Using the OTel telemetry provider will give you insights about specific handlers 
```

Comment:
This documentation is a little confusing here I think since I think the "Otel telemetry provider" is the SDK concept.   But maybe worth mentioning that using these later versions of the gem w/ this will give more detailed internal spans.